node/src/rpc.rs

```diff
@@ -41,7 +41,6 @@ where
 		+ Send
 		+ Sync
 		+ 'static,
-	C: CallApiAt<Block>,
```

Comment:
What is the reason for getting rid of this?

Reply:
It's not a required constrain. For instance in the main [node's implementation](https://github.com/substrate-developer-hub/substrate-node-template/blob/main/node/src/rpc.rs) it is not included.